rosidl_typesupport_cpp/package.xml

```diff
@@ -9,6 +9,7 @@
 
   <buildtool_depend>ament_cmake_ros</buildtool_depend>
 
+  <build_depend>rcpputils</build_depend>
```

Comment:
`rcpputils` also needs to be available at runtime since it provides a shared library. Therefore this should use the `depend` tag instead.

Reply:
Done.

Reply:
Where? It still is only a `build_depend` in this line.

Reply:
Ah, only did it for the C package in 2c29cb28. Will update.

Reply:
Done in e8b0dd3